common/libs/wallet/IWallet.js

```diff
@@ -1,6 +1,7 @@
+// @flow
 import type { RawTransaction } from 'libs/transaction';
 
-export interface IWallet {
+export type IWallet = {
```

Comment:
Why did this get changed? It looks like things that were `: IWallet` before are `: any` now, and there's not type-checking for new wallet classes to ensure they implement certain methods.